src/game/inv.c

```diff
@@ -280,11 +280,13 @@ s32 invAddOneIfCantHaveSlayer(s32 index)
 {
 	if (mainGetStageNum());
 
+#ifdef PLATFORM_N64
 	if (mainGetStageNum() != STAGE_ATTACKSHIP
 			&& mainGetStageNum() != STAGE_SKEDARRUINS
```

Comment:
On the N64 there are 2 exception to the "no Slayer for All Guns" rule which is these 2 stages. Let's make it so we get the behavior for ALL levels.